srml/grandpa/Cargo.toml

```diff
@@ -35,3 +35,4 @@ std = [
 	"session/std",
 	"finality-tracker/std",
 ]
+migrate-authorities = []
```

Comment:
should it not be default?

Reply:
I figured since new chains would want this off that it shouldn't be the default, but I could see it the other way as well.

Reply:
well, we just have to be careful when backporting then. i'd add something to the PR notes for backporters

Reply:
Nit: missing newline at EOF